tapis-project/tapis-ui
# task/TUI-224 - file transfers

## Overview:

File Transfers modal

## Related Github Issues:

- [TUI-224](https://github.com/tapis-project/tapis-ui/issues/224)

## Summary of Changes:

- Refactored tapis-app/_components/Tabs components
- File Transfer apis and hooks
- File Transfer UI elements (TransferCreate, TransferListing, TransferDetails, TransferCancel)
- Modal for File Transfers
- Minor icon and style changes

## Testing Steps:

1. Start transfers
2. Make sure you can see the transfer

## UI Photos:

## Notes:

- The transfer listing does not update on its own. Short file transfers will result in changed statuses when clicking on their details, but the list will not have updated. This has been added to https://github.com/tapis-project/tapis-ui/issues/227
